src/components/stepforms/widgets/CodeEditorWidget.vue

```diff
@@ -38,12 +38,15 @@ export default class CodeEditorWidget extends Mixins(FormWidget) {
   @Prop({ default: '' })
   value!: string;
 
+  @Prop({ type: String, default: '' })
+  lang!: string;
+
   editedValue = this.value;
 
   isFocused = false;
 
-  CodeEditor = CodeEditor;
   // Code editor is set through a responsive data so it can be change after import
+  CodeEditor: CustomCodeEditor = CodeEditor;
```

Comment:
```suggestion
  codeEditor: CustomCodeEditor = CodeEditor;
```
C'est important de différencier un champ `data` d'un import.

Reply:
Même si derrière c'est un switch sur un type d'Éditeur